### DISCUSSION TITLE
Ali ve Masum İnsanlara Yardım Etmek

### DISCUSSION THREAD

**Discussion by slos8888** [sonsuzus/sonsuzus#435] [Soru Cevap]
Problem: 
Arpasland saldırganlar tarafından kuşatıldı. Şehre bir kamyon giriyor. Sürücü, yükün İranlılardan gelen yiyecek ve ilaç olduğunu iddia ediyor. Ali, Arpasland'daki askerlerden biridir. Kamyondan şüpheleniyor, belki de kuşatma yüzündendir. Bir etiketin ardışık iki rakamının toplamı çift ise ve harfi sesli değilse geçerli olduğunu biliyor. Kamyonun etiketinin geçerli olup olmadığını belirleyin.

Bu problem için "A", "E", "I", "O", "U", "Y" harflerini sesli harf olarak kabul ediyoruz.

Input Biçimi:
İlk satır 9 uzunluğunda bir dize içerir. Biçimi "DDXDDD-DD"dir; burada D (sıfır olmayan) bir rakamı temsil eder ve X büyük harfli bir İngilizce harftir.

Output Biçimi:
Etiket geçerliyse "geçerli"  yazdırın (tırnak işaretleri olmadan) aksi taktirde "geçersiz" yazdırın tırnak işaretleri olmadan) 

Link:https://www.hackerearth.com/practice/basic-programming/input-output/basics-of-input-output/practice-problems/algorithm/cartag-948c2b02/

**Comment by sonsuzus** [comment]
Bu soru daha önce eklendi.

**Reply by slos8888** [reply]
Tamamdır hocam görmemişim bugün eklerim yeni bir tane
